homeassistant/components/automation/logbook.py

```diff
@@ -12,9 +12,12 @@ def async_describe_events(hass, async_describe_event):  # type: ignore
     @callback
     def async_describe_logbook_event(event):  # type: ignore
         """Describe a logbook event."""
+        message = "has been triggered"
+        if ATTR_SOURCE in event.data:
+            message = f"{message} by {event.data[ATTR_SOURCE]}"
         return {
             "name": event.data.get(ATTR_NAME),
-            "message": "has been triggered",
+            "message": message,
             "entity_id": event.data.get(ATTR_ENTITY_ID),
```

Comment:
We use the 'message' for when the automation fires for the initial event.

`Watching TV in Master Bedroom is Switched On` has been triggered
becomes
`Watching TV in Master Bedroom is Switched On` has been triggered by `SOURCE`

We currently only consume 'name' in logbook when looking up contexts.  
`Masterbed TV Cabinet 32 Relay #1` turned on by `Watching TV in Master Bedroom is Switch On`

We might want to consume the source in the future, so maybe add


```suggestion
            "source": event.data.get(ATTR_SOURCE),
            "entity_id": event.data.get(ATTR_ENTITY_ID),
```

Reply:
I didn't follow that completely. Also, when I started on this I couldn't find where the `dict` returned by the function registered by calling `async_describe_event` was defined, so just kept to the keys already being used. But, yeah, I'll go ahead and commit your suggested change. 😃